.github/workflows/README.md

```diff
@@ -7,22 +7,18 @@ The following workflows can be found here:
 
 ## Helm Release
```

Comment:
Not related to this PR, but the above listing of workflows in this repository is incomplete.

Reply:
True, we forgot to continue the list. @disrupted can you complete the list in this PR please?